project/settings.py

```diff
@@ -320,10 +320,11 @@ def abspath(*args):
 
 ELASTICSEARCH_HOST = 'http://localhost:9200'
 UNICORE_DISTRIBUTE_HOST = 'http://localhost:6543'
+SERVICE_HOST_IP = '127.0.0.1'
 
 MESOS_MARATHON_HOST = 'http://localhost:8080'
 MESOS_DEFAULT_CPU_SHARE = 0.1
-MESOS_DEFAULT_MEMORY_ALLOCATION = 50.0
+MESOS_DEFAULT_MEMORY_ALLOCATION = 100.0
```

Comment:
The docker runner is strict on resources so less than 100M is not enough since the apps start up at about 90M